SU2_CFD/src/solvers/CIncNSSolver.cpp

```diff
@@ -97,6 +98,43 @@ void CIncNSSolver::Preprocessing(CGeometry *geometry, CSolver **solver_container
 
   ComputeVorticityAndStrainMag<1>(*config, iMesh);
 
+  /*--- Compute recovered pressure and temperature for streamwise periodic flow ---*/
+  if (config->GetKind_Streamwise_Periodic() != NONE) {
+
+    /*--- Define and initialize helping variables ---*/
+    su2double dot_product, Pressure_Recovered, Temperature_Recovered;
```

Comment:
local to the loop please

Reply:
👍 Pressure_Recovered is also now const, the other two not